src/medgen2obo.pl

```diff
@@ -82,7 +82,7 @@
     chomp;
     my ($u,$c) = split(/\t/,$_);
     $uh{$c} = $u;
-    $th{$c}->{xrefs}->{"MEDGEN:$u"} = 1;
+    $th{$c}->{xrefs}->{"Medgen_UID:$u"} = 1;
```

Comment:
## Medgen UID prefix updated
Chris' original ingest only used `UMLS:`. We decided to add `UMLS:` and/or `Medgen:` depending on if the ID started with `C` or `CN`. But there is a 3rd type of ID called the Medgen UID. I believe that Megan said these are for Medgen's internal use and we can ignore them. However I'm going to preserve them for now, and just rename the prefix from what Chris previously had `MEDGEN:` to `Medgen_UID`.

Reply:
Yes, lets keep them.  and you can keep calling them `MEDGEN:`. We should have these three prefix spaces:

MEDGEN:
MEDGEN_CUI:
UMLS:

after we are done.

Reply:
This is a little outdated. I believe we decided UIDs are now in the "Medgen" namespace. And there is also a "MedGenCUI" namespace for the CNs.